website/content/docs/api-clients/cli.mdx

```diff
@@ -34,7 +34,6 @@ The following shows use of the [`-addr`](/boundary/docs/api-clients/cli#addr) fl
 $ boundary authenticate password \
     -addr=https://boundary.example.com:9200
     -auth-method-id=ampw_1234567890 \
```

Comment:
The trailing `\` needs to be removed from this line, and it looks like the prior line is missing one

```suggestion
$ boundary authenticate password \
    -addr=https://boundary.example.com:9200 \
    -auth-method-id=ampw_1234567890
```